packages/astro/components/Code.astro

```diff
@@ -58,6 +58,7 @@ const {
 	experimentalThemes = {},
 	wrap = false,
 	inline = false,
+	...additionalProps
```

Comment:
I think you need to also update the Props type to extend `HTMLAttributed<'some tag here'>`

Reply:
Hey I've tried to add this in. I used `any` for the tag. I'm not sure what would be a correct tag here.